README.md

```diff
@@ -70,6 +63,106 @@ http://localhost:28111 to talk to the faucet of the second node instead.
 
 To copy your Metamask address click on the address at the top of the Metamask panel.
 
+## Preconfirmations
+
+If you tried the demo as instructed above, using the RPC nodes at ports 18126 and 28126, you may
+notice some latency (perhaps around 30s) between confirming a transaction in Metamask and having the
+transaction complete. In large part, this latency is because of the path the transaction has to
+follow between being submitted and being finalized. Refer to
+[the architecture diagram](doc/architecture_phase2.svg) and you'll see that, after a transaction has
+been sequenced, the HotShot Commitment Service must pick it up and send it to the L1 blockchain, it
+must be included in the L1, which can be slow, the L2 node must pick it up from the HotShot
+Contract, and only then can the L2 node fetch your transaction from the sequencer query service,
+execute it, and report the results to Metamask.
+
+This long round trip through the L1 slows down the transaction a lot, especially on a real L1 like
+Ethereum, with a block time of 12 seconds and a finality time of 12 minutes. (For the demo, we use a
+block time of 1 second, and the latency is still noticeable!) One of the advantages of using a
+decentralized consensus protocol as the sequencer is that it can provide _preconfirmations_:
+ahead-of-time assurances that your transaction will be included in the rollup, before the L1 has
```

Comment:
Maybe throw in a disclaimer about inclusion vs execution? This is a point that seems to confuse a lot of people

Reply:
Done